apps/emqx_authn/src/emqx_authn_api.erl

```diff
@@ -99,14 +98,14 @@ paths() ->
         "/authentication",
         "/authentication/:id",
         "/authentication/:id/status",
-        "/authentication/:id/move",
```

Comment:
nit: Do we need to be compatible with the old path of the API?

Reply:
Yeah, I'm trying to find out

Reply:
I synched with @ysfscream - the plan is to update the dashboard in time before the next release, i.e. get this one merged and they'll fix the dashboard in time